community/community-engines/flink-batch/src/test/resources/conf/flink-conf.yaml

```diff
@@ -0,0 +1,67 @@
+################################################################################
```

Comment:
this seems to be getting erased during cross compilation, and never reproduced...

```[INFO] Building - Mahout Community Supported Engines 14.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ mahout-community-engines ---
[INFO]
[INFO] --- apache-rat-plugin:0.13:check (default) @ mahout-community-engines ---
[INFO] Added 2 additional default licenses.
[INFO] Enabled default license matchers.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 64 implicit excludes (use -debug for more details).
[INFO] 20 explicit excludes (use -debug for more details).
[INFO] 1 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 0, unknown: 0, generated: 0, approved: 1 licenses.
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ mahout-community-engines ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-versions) @ mahout-community-engines ---
[INFO]
[INFO] --- maven-remote-resources-plugin:1.7.0:process (process-resource-bundles) @ mahout-community-engines ---
[INFO] Preparing remote bundle org.apache:apache-jar-resource-bundle:1.4
[INFO] Copying 3 resources from 1 bundle.
[INFO]
[INFO] --- maven-site-plugin:3.7.1:attach-descriptor (attach-descriptor) @ mahout-community-engines ---
[INFO] No site descriptor found: nothing to attach.
[INFO]
[INFO] --- apache-rat-plugin:0.13:check (default) @ mahout-community-engines ---
[INFO] Added 2 additional default licenses.
[INFO] Enabled default license matchers.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 64 implicit excludes (use -debug for more details).
[INFO] 20 explicit excludes (use -debug for more details).
[INFO] 1 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 0, unknown: 0, generated: 0, approved: 1 licenses.
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ mahout-community-engines ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-versions) @ mahout-community-engines ---
[INFO]
[INFO] --- maven-remote-resources-plugin:1.7.0:process (process-resource-bundles) @ mahout-community-engines ---
[INFO] Preparing remote bundle org.apache:apache-jar-resource-bundle:1.4
[INFO] Copying 3 resources from 1 bundle.
[INFO]
[INFO] --- maven-site-plugin:3.7.1:attach-descriptor (attach-descriptor) @ mahout-community-engines ---
[INFO] No site descriptor found: nothing to attach.
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ mahout-community-engines ---
[INFO] Installing /Users/colleenpalumbo/sandbox/mahout/community/community-engines/pom.xml to /Users/colleenpalumbo/.m2/repository/org/apache/mahout/mahout-community-engines/14.1-SNAPSHOT/mahout-community-engines-14.1-SNAPSHOT.pom
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building -- Mahout Flink Engine (Community) 14.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-runtime_2.11/1.10.0/flink-runtime_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-runtime_2.11/1.10.0/flink-runtime_2.11-1.10.0.pom (21 KB at 129.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-parent/1.10.0/flink-parent-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-parent/1.10.0/flink-parent-1.10.0.pom (63 KB at 543.0 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/apache/20/apache-20.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/apache/20/apache-20.pom (16 KB at 276.4 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-core/1.10.0/flink-core-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-core/1.10.0/flink-core-1.10.0.pom (11 KB at 154.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-annotations/1.10.0/flink-annotations-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-annotations/1.10.0/flink-annotations-1.10.0.pom (4 KB at 89.0 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/force-shading/1.10.0/force-shading-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/force-shading/1.10.0/force-shading-1.10.0.pom (4 KB at 64.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics-core/1.10.0/flink-metrics-core-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics-core/1.10.0/flink-metrics-core-1.10.0.pom (5 KB at 78.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics/1.10.0/flink-metrics-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics/1.10.0/flink-metrics-1.10.0.pom (3 KB at 47.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-asm-7/7.1-9.0/flink-shaded-asm-7-7.1-9.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-asm-7/7.1-9.0/flink-shaded-asm-7-7.1-9.0.pom (2 KB at 33.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded/9.0/flink-shaded-9.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded/9.0/flink-shaded-9.0.pom (12 KB at 256.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/esotericsoftware/kryo/kryo/2.24.0/kryo-2.24.0.pom
Downloaded: https://repo.maven.apache.org/maven2/com/esotericsoftware/kryo/kryo/2.24.0/kryo-2.24.0.pom (7 KB at 76.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/esotericsoftware/minlog/minlog/1.2/minlog-1.2.pom
Downloaded: https://repo.maven.apache.org/maven2/com/esotericsoftware/minlog/minlog/1.2/minlog-1.2.pom (2 KB at 36.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/objenesis/objenesis/2.1/objenesis-2.1.pom
Downloaded: https://repo.maven.apache.org/maven2/org/objenesis/objenesis/2.1/objenesis-2.1.pom (3 KB at 49.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/objenesis/objenesis-parent/2.1/objenesis-parent-2.1.pom
Downloaded: https://repo.maven.apache.org/maven2/org/objenesis/objenesis-parent/2.1/objenesis-parent-2.1.pom (17 KB at 339.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-guava/18.0-9.0/flink-shaded-guava-18.0-9.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-guava/18.0-9.0/flink-shaded-guava-18.0-9.0.pom (2 KB at 33.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-java/1.10.0/flink-java-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-java/1.10.0/flink-java-1.10.0.pom (6 KB at 99.0 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-queryable-state-client-java/1.10.0/flink-queryable-state-client-java-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-queryable-state-client-java/1.10.0/flink-queryable-state-client-java-1.10.0.pom (8 KB at 148.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-queryable-state/1.10.0/flink-queryable-state-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-queryable-state/1.10.0/flink-queryable-state-1.10.0.pom (2 KB at 47.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-netty/4.1.39.Final-9.0/flink-shaded-netty-4.1.39.Final-9.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-netty/4.1.39.Final-9.0/flink-shaded-netty-4.1.39.Final-9.0.pom (3 KB at 55.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-fs/1.10.0/flink-hadoop-fs-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-fs/1.10.0/flink-hadoop-fs-1.10.0.pom (9 KB at 222.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-filesystems/1.10.0/flink-filesystems-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-filesystems/1.10.0/flink-filesystems-1.10.0.pom (3 KB at 41.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-jackson/2.10.1-9.0/flink-shaded-jackson-2.10.1-9.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-jackson/2.10.1-9.0/flink-shaded-jackson-2.10.1-9.0.pom (2 KB at 28.4 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-jackson-parent/2.10.1-9.0/flink-shaded-jackson-parent-2.10.1-9.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-jackson-parent/2.10.1-9.0/flink-shaded-jackson-parent-2.10.1-9.0.pom (4 KB at 47.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/javassist/javassist/3.24.0-GA/javassist-3.24.0-GA.pom
Downloaded: https://repo.maven.apache.org/maven2/org/javassist/javassist/3.24.0-GA/javassist-3.24.0-GA.pom (11 KB at 187.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-actor_2.11/2.5.21/akka-actor_2.11-2.5.21.pom
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-actor_2.11/2.5.21/akka-actor_2.11-2.5.21.pom (3 KB at 30.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/config/1.3.3/config-1.3.3.pom
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/config/1.3.3/config-1.3.3.pom (2 KB at 29.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/scala-lang/modules/scala-java8-compat_2.11/0.7.0/scala-java8-compat_2.11-0.7.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/scala-lang/modules/scala-java8-compat_2.11/0.7.0/scala-java8-compat_2.11-0.7.0.pom (3 KB at 58.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-stream_2.11/2.5.21/akka-stream_2.11-2.5.21.pom
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-stream_2.11/2.5.21/akka-stream_2.11-2.5.21.pom (3 KB at 57.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-protobuf_2.11/2.5.21/akka-protobuf_2.11-2.5.21.pom
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-protobuf_2.11/2.5.21/akka-protobuf_2.11-2.5.21.pom (2 KB at 32.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/reactivestreams/reactive-streams/1.0.2/reactive-streams-1.0.2.pom
Downloaded: https://repo.maven.apache.org/maven2/org/reactivestreams/reactive-streams/1.0.2/reactive-streams-1.0.2.pom (2 KB at 19.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/ssl-config-core_2.11/0.3.7/ssl-config-core_2.11-0.3.7.pom
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/ssl-config-core_2.11/0.3.7/ssl-config-core_2.11-0.3.7.pom (4 KB at 50.4 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.11/1.1.1/scala-parser-combinators_2.11-1.1.1.pom
Downloaded: https://repo.maven.apache.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.11/1.1.1/scala-parser-combinators_2.11-1.1.1.pom (3 KB at 42.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-slf4j_2.11/2.5.21/akka-slf4j_2.11-2.5.21.pom
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-slf4j_2.11/2.5.21/akka-slf4j_2.11-2.5.21.pom (3 KB at 42.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/clapper/grizzled-slf4j_2.11/1.3.2/grizzled-slf4j_2.11-1.3.2.pom
Downloaded: https://repo.maven.apache.org/maven2/org/clapper/grizzled-slf4j_2.11/1.3.2/grizzled-slf4j_2.11-1.3.2.pom (2 KB at 41.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/github/scopt/scopt_2.11/3.7.1/scopt_2.11-3.7.1.pom
Downloaded: https://repo.maven.apache.org/maven2/com/github/scopt/scopt_2.11/3.7.1/scopt_2.11-3.7.1.pom (3 KB at 40.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/xerial/snappy/snappy-java/1.1.4/snappy-java-1.1.4.pom
Downloaded: https://repo.maven.apache.org/maven2/org/xerial/snappy/snappy-java/1.1.4/snappy-java-1.1.4.pom (4 KB at 64.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/twitter/chill_2.11/0.7.6/chill_2.11-0.7.6.pom
Downloaded: https://repo.maven.apache.org/maven2/com/twitter/chill_2.11/0.7.6/chill_2.11-0.7.6.pom (3 KB at 48.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/twitter/chill-java/0.7.6/chill-java-0.7.6.pom
Downloaded: https://repo.maven.apache.org/maven2/com/twitter/chill-java/0.7.6/chill-java-0.7.6.pom (2 KB at 3.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/lz4/lz4-java/1.5.0/lz4-java-1.5.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/lz4/lz4-java/1.5.0/lz4-java-1.5.0.pom (2 KB at 48.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-scala_2.11/1.10.0/flink-scala_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-scala_2.11/1.10.0/flink-scala_2.11-1.10.0.pom (13 KB at 280.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-compatibility_2.11/1.10.0/flink-hadoop-compatibility_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-compatibility_2.11/1.10.0/flink-hadoop-compatibility_2.11-1.10.0.pom (16 KB at 228.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-connectors/1.10.0/flink-connectors-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-connectors/1.10.0/flink-connectors-1.10.0.pom (5 KB at 38.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-clients_2.11/1.10.0/flink-clients_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-clients_2.11/1.10.0/flink-clients_2.11-1.10.0.pom (9 KB at 64.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-optimizer_2.11/1.10.0/flink-optimizer_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-optimizer_2.11/1.10.0/flink-optimizer_2.11-1.10.0.pom (6 KB at 90.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils_2.11/1.10.0/flink-test-utils_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils_2.11/1.10.0/flink-test-utils_2.11-1.10.0.pom (7 KB at 50.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils-parent/1.10.0/flink-test-utils-parent-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils-parent/1.10.0/flink-test-utils-parent-1.10.0.pom (2 KB at 28.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils-junit/1.10.0/flink-test-utils-junit-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils-junit/1.10.0/flink-test-utils-junit-1.10.0.pom (4 KB at 88.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-streaming-java_2.11/1.10.0/flink-streaming-java_2.11-1.10.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-streaming-java_2.11/1.10.0/flink-streaming-java_2.11-1.10.0.pom (6 KB at 100.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/curator/curator-test/2.12.0/curator-test-2.12.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/curator/curator-test/2.12.0/curator-test-2.12.0.pom (5 KB at 88.0 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/curator/apache-curator/2.12.0/apache-curator-2.12.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/curator/apache-curator/2.12.0/apache-curator-2.12.0.pom (32 KB at 657.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-tests_2.11/1.8.3/flink-tests_2.11-1.8.3.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-tests_2.11/1.8.3/flink-tests_2.11-1.8.3.pom (41 KB at 261.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-parent/1.8.3/flink-parent-1.8.3.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-parent/1.8.3/flink-parent-1.8.3.pom (59 KB at 424.0 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-guava/18.0-6.0/flink-shaded-guava-18.0-6.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-guava/18.0-6.0/flink-shaded-guava-18.0-6.0.pom (2 KB at 47.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded/6.0/flink-shaded-6.0.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded/6.0/flink-shaded-6.0.pom (10 KB at 234.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/force-shading/1.8.3/force-shading-1.8.3.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/force-shading/1.8.3/force-shading-1.8.3.pom (4 KB at 67.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-runtime_2.11/1.10.0/flink-runtime_2.11-1.10.0.jar
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-queryable-state-client-java/1.10.0/flink-queryable-state-client-java-1.10.0.jar
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-fs/1.10.0/flink-hadoop-fs-1.10.0.jar
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-netty/4.1.39.Final-9.0/flink-shaded-netty-4.1.39.Final-9.0.jar
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-guava/18.0-9.0/flink-shaded-guava-18.0-9.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-fs/1.10.0/flink-hadoop-fs-1.10.0.jar (38 KB at 378.3 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-asm-7/7.1-9.0/flink-shaded-asm-7-7.1-9.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-queryable-state-client-java/1.10.0/flink-queryable-state-client-java-1.10.0.jar (84 KB at 625.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-jackson/2.10.1-9.0/flink-shaded-jackson-2.10.1-9.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-asm-7/7.1-9.0/flink-shaded-asm-7-7.1-9.0.jar (278 KB at 660.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/javassist/javassist/3.24.0-GA/javassist-3.24.0-GA.jar
Downloaded: https://repo.maven.apache.org/maven2/org/javassist/javassist/3.24.0-GA/javassist-3.24.0-GA.jar (760 KB at 509.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-actor_2.11/2.5.21/akka-actor_2.11-2.5.21.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-netty/4.1.39.Final-9.0/flink-shaded-netty-4.1.39.Final-9.0.jar (4191 KB at 1399.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/config/1.3.3/config-1.3.3.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-guava/18.0-9.0/flink-shaded-guava-18.0-9.0.jar (2372 KB at 770.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/scala-lang/modules/scala-java8-compat_2.11/0.7.0/scala-java8-compat_2.11-0.7.0.jar
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/config/1.3.3/config-1.3.3.jar (280 KB at 84.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-stream_2.11/2.5.21/akka-stream_2.11-2.5.21.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-jackson/2.10.1-9.0/flink-shaded-jackson-2.10.1-9.0.jar (2321 KB at 618.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/reactivestreams/reactive-streams/1.0.2/reactive-streams-1.0.2.jar
Downloaded: https://repo.maven.apache.org/maven2/org/reactivestreams/reactive-streams/1.0.2/reactive-streams-1.0.2.jar (3 KB at 0.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/ssl-config-core_2.11/0.3.7/ssl-config-core_2.11-0.3.7.jar
Downloaded: https://repo.maven.apache.org/maven2/org/scala-lang/modules/scala-java8-compat_2.11/0.7.0/scala-java8-compat_2.11-0.7.0.jar (569 KB at 149.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-protobuf_2.11/2.5.21/akka-protobuf_2.11-2.5.21.jar
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/ssl-config-core_2.11/0.3.7/ssl-config-core_2.11-0.3.7.jar (393 KB at 90.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-slf4j_2.11/2.5.21/akka-slf4j_2.11-2.5.21.jar
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-slf4j_2.11/2.5.21/akka-slf4j_2.11-2.5.21.jar (22 KB at 4.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/clapper/grizzled-slf4j_2.11/1.3.2/grizzled-slf4j_2.11-1.3.2.jar
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-protobuf_2.11/2.5.21/akka-protobuf_2.11-2.5.21.jar (474 KB at 106.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/github/scopt/scopt_2.11/3.7.1/scopt_2.11-3.7.1.jar
Downloaded: https://repo.maven.apache.org/maven2/org/clapper/grizzled-slf4j_2.11/1.3.2/grizzled-slf4j_2.11-1.3.2.jar (8 KB at 1.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/xerial/snappy/snappy-java/1.1.4/snappy-java-1.1.4.jar
Downloaded: https://repo.maven.apache.org/maven2/com/github/scopt/scopt_2.11/3.7.1/scopt_2.11-3.7.1.jar (152 KB at 32.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/twitter/chill_2.11/0.7.6/chill_2.11-0.7.6.jar
Downloaded: https://repo.maven.apache.org/maven2/com/twitter/chill_2.11/0.7.6/chill_2.11-0.7.6.jar (222 KB at 44.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/twitter/chill-java/0.7.6/chill-java-0.7.6.jar
Downloaded: https://repo.maven.apache.org/maven2/com/twitter/chill-java/0.7.6/chill-java-0.7.6.jar (59 KB at 11.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/lz4/lz4-java/1.5.0/lz4-java-1.5.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/lz4/lz4-java/1.5.0/lz4-java-1.5.0.jar (500 KB at 83.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/force-shading/1.10.0/force-shading-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/force-shading/1.10.0/force-shading-1.10.0.jar (8 KB at 1.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-scala_2.11/1.10.0/flink-scala_2.11-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/xerial/snappy/snappy-java/1.1.4/snappy-java-1.1.4.jar (1471 KB at 219.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-compatibility_2.11/1.10.0/flink-hadoop-compatibility_2.11-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-actor_2.11/2.5.21/akka-actor_2.11-2.5.21.jar (3928 KB at 579.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-java/1.10.0/flink-java-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-hadoop-compatibility_2.11/1.10.0/flink-hadoop-compatibility_2.11-1.10.0.jar (79 KB at 11.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-core/1.10.0/flink-core-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-scala_2.11/1.10.0/flink-scala_2.11-1.10.0.jar (847 KB at 118.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-annotations/1.10.0/flink-annotations-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-annotations/1.10.0/flink-annotations-1.10.0.jar (16 KB at 2.2 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics-core/1.10.0/flink-metrics-core-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-java/1.10.0/flink-java-1.10.0.jar (428 KB at 58.9 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/esotericsoftware/kryo/kryo/2.24.0/kryo-2.24.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-metrics-core/1.10.0/flink-metrics-core-1.10.0.jar (19 KB at 2.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/com/esotericsoftware/minlog/minlog/1.2/minlog-1.2.jar
Downloaded: https://repo.maven.apache.org/maven2/com/esotericsoftware/minlog/minlog/1.2/minlog-1.2.jar (5 KB at 0.7 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-clients_2.11/1.10.0/flink-clients_2.11-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-clients_2.11/1.10.0/flink-clients_2.11-1.10.0.jar (130 KB at 17.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-optimizer_2.11/1.10.0/flink-optimizer_2.11-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/com/esotericsoftware/kryo/kryo/2.24.0/kryo-2.24.0.jar (332 KB at 43.1 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils_2.11/1.10.0/flink-test-utils_2.11-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-runtime_2.11/1.10.0/flink-runtime_2.11-1.10.0.jar (11728 KB at 1485.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils-junit/1.10.0/flink-test-utils-junit-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/com/typesafe/akka/akka-stream_2.11/2.5.21/akka-stream_2.11-2.5.21.jar (5288 KB at 667.4 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-runtime_2.11/1.10.0/flink-runtime_2.11-1.10.0-tests.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils-junit/1.10.0/flink-test-utils-junit-1.10.0.jar (31 KB at 3.8 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-streaming-java_2.11/1.10.0/flink-streaming-java_2.11-1.10.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-optimizer_2.11/1.10.0/flink-optimizer_2.11-1.10.0.jar (352 KB at 43.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/curator/curator-test/2.12.0/curator-test-2.12.0.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-core/1.10.0/flink-core-1.10.0.jar (1537 KB at 180.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/zookeeper/zookeeper/3.4.8/zookeeper-3.4.8.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-streaming-java_2.11/1.10.0/flink-streaming-java_2.11-1.10.0.jar (1024 KB at 115.6 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/jline/jline/0.9.94/jline-0.9.94.jar
Downloaded: https://repo.maven.apache.org/maven2/jline/jline/0.9.94/jline-0.9.94.jar (86 KB at 9.5 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/io/netty/netty/3.7.0.Final/netty-3.7.0.Final.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-test-utils_2.11/1.10.0/flink-test-utils_2.11-1.10.0.jar (1415 KB at 154.4 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/flink/flink-tests_2.11/1.8.3/flink-tests_2.11-1.8.3.jar
Downloaded: https://repo.maven.apache.org/maven2/org/apache/zookeeper/zookeeper/3.4.8/zookeeper-3.4.8.jar (786 KB at 84.3 KB/sec)
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-tests_2.11/1.8.3/flink-tests_2.11-1.8.3.jar (10 KB at 1.0 KB/sec)
Downloaded: https://repo.maven.apache.org/maven2/io/netty/netty/3.7.0.Final/netty-3.7.0.Final.jar (1181 KB at 118.6 KB/sec)
Downloaded: https://repo.maven.apache.org/maven2/org/apache/curator/curator-test/2.12.0/curator-test-2.12.0.jar (2368 KB at 232.6 KB/sec)
Downloaded: https://repo.maven.apache.org/maven2/org/apache/flink/flink-runtime_2.11/1.10.0/flink-runtime_2.11-1.10.0-tests.jar (3691 KB at 352.7 KB/sec)
[INFO]
[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ mahout-flink-batch ---
[INFO]
[INFO] --- apache-rat-plugin:0.13:check (default) @ mahout-flink-batch ---
[INFO] Added 2 additional default licenses.
[INFO] Enabled default license matchers.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 62 implicit excludes (use -debug for more details).
[INFO] 20 explicit excludes (use -debug for more details).
[INFO] 40 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 0, unknown: 0, generated: 0, approved: 40 licenses.
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ mahout-flink-batch ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-versions) @ mahout-flink-batch ---
[INFO]
[INFO] --- maven-remote-resources-plugin:1.7.0:process (process-resource-bundles) @ mahout-flink-batch ---
[INFO] Preparing remote bundle org.apache:apache-jar-resource-bundle:1.4
[INFO] Copying 3 resources from 1 bundle.
[INFO]
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ mahout-flink-batch ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 3 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ mahout-flink-batch ---
[INFO] No sources to compile
[INFO]
[INFO] --- scala-maven-plugin:4.3.1:compile (scala-compile) @ mahout-flink-batch ---
[INFO] Using incremental compilation using Mixed compile order
[INFO] Compiler bridge file: /Users/colleenpalumbo/.sbt/1.0/zinc/org.scala-sbt/org.scala-sbt-compiler-bridge_2.11-1.3.2-bin_2.11.12__52.0-1.3.2_20200115T025827.jar
[INFO] Compiling 23 Scala sources to /Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/classes ...
[WARNING] [Warn] : there were two deprecation warnings; re-run with -deprecation for details
[WARNING] [Warn] : there were 12 feature warnings; re-run with -feature for details
[WARNING] two warnings found
[INFO] Done compiling.
[INFO] compile in 14.3 s
[INFO]
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ mahout-flink-batch ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 3 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ mahout-flink-batch ---
[INFO] No sources to compile
[INFO]
[INFO] --- scala-maven-plugin:4.3.1:testCompile (scala-test-compile) @ mahout-flink-batch ---
[INFO] Using incremental compilation using Mixed compile order
[INFO] Compiler bridge file: /Users/colleenpalumbo/.sbt/1.0/zinc/org.scala-sbt/org.scala-sbt-compiler-bridge_2.11-1.3.2-bin_2.11.12__52.0-1.3.2_20200115T025827.jar
[INFO] Compiling 16 Scala sources to /Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/test-classes ...
[WARNING] [Warn] : there were 15 deprecation warnings; re-run with -deprecation for details
[WARNING] [Warn] : there was one feature warning; re-run with -feature for details
[WARNING] two warnings found
[INFO] Done compiling.
[INFO] compile in 10.0 s
[INFO]
[INFO] --- maven-surefire-plugin:3.0.0-M4:test (default-test) @ mahout-flink-batch ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- scalatest-maven-plugin:2.0.0:test (scala-test) @ mahout-flink-batch ---
[INFO] Forking ScalaTest via: /bin/sh -c cd '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch' && 'java' '-Djava.home=/Library/Java/JavaVirtualMachines/jdk1.8.0_201.jdk/Contents/Home/jre' '-Djdk.lang.Process.launchMechanism=FORK' '-Dmahout.home=/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/test-classes' '-Dbasedir=/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch' '-Xmx4g' 'org.scalatest.tools.Runner' '-R' '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/classes /Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/test-classes' '-o' '-fWDF' '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/surefire-reports/TestSuite.txt' '-u' '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/surefire-reports/.'
Discovery starting.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Discovery completed in 418 milliseconds.
Run starting. Expected test count is: 114
DrmLikeOpsSuite:
org.apache.mahout.flinkbindings.DrmLikeOpsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.DrmLikeOpsSuite.initContext(DrmLikeOpsSuite.scala:30)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.DrmLikeOpsSuite.beforeAll(DrmLikeOpsSuite.scala:30)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
LATestSuite:
org.apache.mahout.flinkbindings.blas.LATestSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.blas.LATestSuite.initContext(LATestSuite.scala:31)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.blas.LATestSuite.beforeAll(LATestSuite.scala:31)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
UseCasesSuite:
org.apache.mahout.flinkbindings.UseCasesSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.UseCasesSuite.initContext(UseCasesSuite.scala:32)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.UseCasesSuite.beforeAll(UseCasesSuite.scala:32)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
FlinkDistributedDecompositionsSuite:
org.apache.mahout.flinkbindings.standard.FlinkDistributedDecompositionsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.FlinkDistributedDecompositionsSuite.initContext(FlinkDistributedDecompositionsSuite.scala:38)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.FlinkDistributedDecompositionsSuite.beforeAll(FlinkDistributedDecompositionsSuite.scala:38)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
ClusteringSuite:
org.apache.mahout.flinkbindings.standard.ClusteringSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.ClusteringSuite.initContext(ClusteringSuite.scala:24)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.ClusteringSuite.beforeAll(ClusteringSuite.scala:24)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
PreprocessorSuite:
org.apache.mahout.flinkbindings.standard.PreprocessorSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.PreprocessorSuite.initContext(PreprocessorSuite.scala:25)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.PreprocessorSuite.beforeAll(PreprocessorSuite.scala:25)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
RLikeOpsSuite:
org.apache.mahout.flinkbindings.RLikeOpsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.RLikeOpsSuite.initContext(RLikeOpsSuite.scala:30)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.RLikeOpsSuite.beforeAll(RLikeOpsSuite.scala:30)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
FlinkByteBCastSuite:
- BCast vector
- BCast matrix
RegressionTestsSuite:
org.apache.mahout.flinkbindings.standard.RegressionTestsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.RegressionTestsSuite.initContext(RegressionTestsSuite.scala:24)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.RegressionTestsSuite.beforeAll(RegressionTestsSuite.scala:24)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
RegressionSuite:
org.apache.mahout.flinkbindings.standard.RegressionSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.RegressionSuite.initContext(RegressionSuite.scala:25)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.RegressionSuite.beforeAll(RegressionSuite.scala:25)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
NaiveBayesTestSuite:
org.apache.mahout.flinkbindings.standard.NaiveBayesTestSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.NaiveBayesTestSuite.initContext(NaiveBayesTestSuite.scala:26)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.NaiveBayesTestSuite.beforeAll(NaiveBayesTestSuite.scala:26)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
DrmLikeSuite:
org.apache.mahout.flinkbindings.standard.DrmLikeSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.DrmLikeSuite.initContext(DrmLikeSuite.scala:25)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.DrmLikeSuite.beforeAll(DrmLikeSuite.scala:25)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
RLikeDrmOpsSuite:
org.apache.mahout.flinkbindings.standard.RLikeDrmOpsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.RLikeDrmOpsSuite.initContext(RLikeDrmOpsSuite.scala:25)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.RLikeDrmOpsSuite.beforeAll(RLikeDrmOpsSuite.scala:25)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
DrmLikeOpsSuite:
org.apache.mahout.flinkbindings.standard.DrmLikeOpsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:94)
  at org.apache.mahout.flinkbindings.FlinkDistributedContext.<init>(FlinkDistributedContext.scala:30)
  at org.apache.mahout.flinkbindings.package$.wrapContext(package.scala:52)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.initContext(DistributedFlinkSuite.scala:48)
  at org.apache.mahout.flinkbindings.standard.DrmLikeOpsSuite.initContext(DrmLikeOpsSuite.scala:25)
  at org.apache.mahout.flinkbindings.DistributedFlinkSuite$class.beforeAll(DistributedFlinkSuite.scala:68)
  at org.apache.mahout.flinkbindings.standard.DrmLikeOpsSuite.beforeAll(DrmLikeOpsSuite.scala:25)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.liftedTree1$1(BeforeAndAfterAllConfigMap.scala:247)
  at org.scalatest.BeforeAndAfterAllConfigMap$class.run(BeforeAndAfterAllConfigMap.scala:245)
  ...
Run completed in 4 seconds, 340 milliseconds.
Total number of tests run: 2
Suites: completed 2, aborted 13
Tests: succeeded 2, failed 0, canceled 0, ignored 0, pending 0
*** 13 SUITES ABORTED ***
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Apache Mahout ...................................... SUCCESS [ 25.692 s]
[INFO] Mahout Core ........................................ SUCCESS [07:16 min]
[INFO] Mahout Engine ...................................... SUCCESS [  0.103 s]
[INFO] - Mahout HDFS Support .............................. SUCCESS [ 20.551 s]
[INFO] - Mahout Spark Engine .............................. SUCCESS [02:44 min]
[INFO] Mahout Community ................................... SUCCESS [  0.175 s]
[INFO] - Mahout Community Supported Engines ............... SUCCESS [  0.065 s]
[INFO] -- Mahout Flink Engine (Community) ................. FAILURE [ 46.078 s]
[INFO] -- Mahout H2O Engine (Community) ................... SKIPPED
[INFO] - Mahout Classic (Map-Reduce Based Machine Learning) SKIPPED
[INFO] -- Mahout Classic: Algorithms and Code ............. SKIPPED
[INFO] -- Mahout Classic Integration ...................... SKIPPED
[INFO] -- Mahout Classic: Examples ........................ SKIPPED
[INFO] - Mahout Spark CLI Drivers ......................... SKIPPED
[INFO] Mahout Release Package ............................. SKIPPED
[INFO] Mahout Examples .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11:34 min
[INFO] Finished at: 2020-04-23T11:49:10-07:00
[INFO] Final Memory: 93M/1054M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.scalatest:scalatest-maven-plugin:2.0.0:test (scala-test) on project mahout-flink-batch: There are test failures -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :mahout-flink-batch
```

Reply:
https://github.com/chrisdutz/mahout/tree/master/conf

missing in `@chrisdutz:master`.

Wow that would be cool if it still passed tests..